tests/rgw/test_s3.py

```diff
@@ -177,7 +177,7 @@ def execute_s3_tests(node: CephNode, build: str, encryption: bool = False) -> in
         extra_args = "-a '!fails_on_rgw,!fails_strict_rfc2616,!encryption'"
         tests = "s3tests"
 
-        if not build.startswith("7"):
+        if not (build.startswith("7") or build.startswith("8")):
```

Comment:
since its needed for version > ceph 7, we might need to update this every time there is new ceph major versions.
So can we update this in such a way this should work for all ceph version >= ceph 7

Reply:
done please review:
http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/cephci-run-APQ6OS/execute_s3tests_0.log